frontend/src/components/object/ObjectVersion.vue

```diff
@@ -126,17 +138,17 @@ watch( getVersions, () => {
                 :to="{ name: RouteNames.DETAIL_OBJECTS,
                        query: { objectId: props.objectId, versionId: data.id } }"
               >
-                {{ data.updatedAt ? formatDateLong(data.updatedAt) : formatDateLong(data.createdAt) }}
+                {{ data.s3VersionId ? formatDateLong(data.createdAt) : formatDateLong(data.createdAt?? data.updatedAt) }}
               </router-link>
               <span v-else>
-                {{ data.updatedAt ? formatDateLong(data.updatedAt) : formatDateLong(data.createdAt) }}
+                {{ data.s3VersionId ? formatDateLong(data.createdAt) : formatDateLong(data.createdAt?? data.updatedAt) }}
```

Comment:
There a reason why we can't collapse this to just the following? Unsure of requirements here, but linting is complaining about string length.
```ts
{{ formatDateLong(data.createdAt ?? data.updatedAt) }}
```